app/assets/stylesheets/papers.scss

```diff
@@ -1,3 +1,120 @@
 // Place all the styles related to the Papers controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+#papers {
+  border-radius: 5px;
+  height: 35px;
+  width: 1115px;
+  background-color: #f2f2f2;
+  margin-left: auto;
+  margin-right: auto;
+  font-size: 15px;
+  color: #8c8c8c;
+  padding-left: 15px;
+  padding-top: 8px;
+}
+
+#table {
+ margin-left: 165px;
+ margin-right: auto;
+ width: 1110px;
+}
+
+#header_names {
+  color: #0074D9;
+}
+
+#button_tool {
+ margin-left: auto;
+ margin-right: auto;
+ width: 190px;
+}
+
+#new_paper_button {
+  top: 100;
+  margin-left: 1000px;
+}
+
+#custom-search-input {
+  padding: 3px;
+  border: solid 1px #E4E4E4;
+  border-radius: 6px;
+  background-color: #fff;
+}
+
+#custom-search-input input {
+  border: 0;
+  box-shadow: none;
+}
+
+#custom-search-input button {
+  margin: 2px 0 0 0;
+  background: none;
+  box-shadow: none;
+  border: 0;
+  color: #666666;
+  padding: 0 8px 0 10px;
+  border-left: solid 1px #ccc;
+}
+
+#custom-search-input button:hover {
+  border: 0;
+  box-shadow: none;
+  border-left: solid 1px #ccc;
+}
+
+#custom-search-input .glyphicon-search {
+  font-size: 23px;
+}
+
+#authors_boxed_list {
+  select,ul { height: 300px; overflow: auto; width: 300px; border: 3px solid #bfbfbf; }
+  ul { list-style-type: none; margin: 0; padding: 0; overflow-x: hidden; }
+  li { margin: 0; padding: 0; }
+  label { display: block; color: WindowText; background-color: transparent; margin: 0; padding: 0; width: 100%; }
+  label:hover { background-color: Highlight; color: HighlightText; }
+}
+
+#authors_boxed_list label input {
+  margin-right: 5px;
+  margin-left: 10px;
+}
+
+.outline {
+  background-color: transparent;
+  color: inherit;
+  transition: all .25s;
+}
+
+.btn-primary.outline {
```

Comment:
Are these repeated from authors.scss? Never repeat yourself in code without a good reason. Move them into the main application.scss if they is appropriate.